src/Spd.Resource.Repository/ControllingMemberCrcApplication/ControllingMemberCrcRepository.cs

```diff
@@ -66,7 +66,7 @@ public async Task<ControllingMemberCrcApplicationCmdResp> CreateControllingMembe
 
         //link to bizContact
         var bizContact = _context.spd_businesscontacts.Where(x => x.spd_businesscontactid == cmd.BizContactId).FirstOrDefault();
-        _context.SetLink(app, nameof(spd_application.spd_businesscontact_spd_application), bizContact);
+        _context.AddLink(bizContact, nameof(spd_application.spd_businesscontact_spd_application), app);
```

Comment:
I am afraid if this is called many times, will bizContact has many app.
for me, one app should only have 0 or 1 bizContact, so, use previous one is after.
besides, if we decide to use AddLink, should line 130 needs change?